pkg/controller/external_secrets/install_external_secrets.go

```diff
@@ -87,6 +87,11 @@ func (r *Reconciler) reconcileExternalSecretsDeployment(esc *operatorv1alpha1.Ex
 		return err
 	}
 
+	if err := r.createOrApplyNetworkPolicies(esc, resourceLabels, recon); err != nil {
```

Comment:
Please follow the installation order defined [here](https://github.com/helm/helm/blob/main/pkg%2Frelease%2Fv1%2Futil%2Fkind_sorter.go#L31)